electrum/gui/qt/my_treeview.py

```diff
@@ -451,6 +452,13 @@ def add_copy_menu(self, menu: QMenu, idx) -> QMenu:
         return cc
 
     def place_text_on_clipboard(self, text: str, *, title: str = None) -> None:
+        if title in {
+            "Amount",
+            "Balance",
+        } or title.endswith(" Value") or title.endswith(" Acquisition price") or title.endswith(" Capital Gains"):
```

Comment:
I expect this would raise if title was not provided (i.e. is `None`).
Also, these strings are often translated, in which case the logic does not make sense...